pkg/placement/placement-resolution.go

```diff
@@ -63,6 +68,11 @@ func (resolution *placementResolution) noteObject(obj runtime.Object) (bool, err
 		return exists, nil
 	}
 
+	if exists {
+		// the object is already in the resolution but something internal changed
```

Comment:
Something _internal_ to _what_? I think it would be clearer to s/something internal/the object content/